Microsoft.Toolkit.Sample.Wpf.App/MainWindow.xaml.cs

```diff
@@ -38,12 +38,24 @@ private void inkToolbar_ActiveToolChanged(object sender, object e)
 
         private async void myMap_Loaded(object sender, RoutedEventArgs e)
         {
-            // Specify a known location.
-            BasicGeoposition cityPosition = new BasicGeoposition() { Latitude = 47.604, Longitude = -122.329 };
-            var cityCenter = new Geopoint(cityPosition);
-
-            // Set the map location.
-            await myMap.TrySetViewAsync(cityCenter, 12).ConfigureAwait(false);
+            if (myMap.MapElements.Count == 0)
```

Comment:
Are you seeing multiple loaded events from the map?
I'm curious about this check, seems odd for a sample and I would encourage you to avoid this.